sortinghat/core/models.py

```diff
@@ -134,34 +134,73 @@ def __str__(self):
         return '%s - %s - %s - %s - %s' % (self.ouid, self.trx, self.op_type, self.entity_type, self.target)
 
 
-class Organization(EntityBase):
+class Group(MP_Node, EntityBase):
+    class GroupType(Enum):
+        organization = 'Organization'
+        team = 'Team'
+
+        @classmethod
+        def choices(cls):
+            return ((item.name, item.value) for item in cls)
+
+        def __str__(self):
+            return self.value
+
     name = CharField(max_length=MAX_SIZE_CHAR_INDEX)
+    type = CharField(max_length=12, choices=GroupType.choices())
+    organization = ForeignKey('self', related_name='teams', on_delete=CASCADE,
```

Comment:
I'm not very sure about this field. It's for teams but not for organizations. The main problem is the name of the field in the table: `organization_id` which is not clear what its purpose is or what it stores.